askama_derive/src/generator/node.rs

```diff
@@ -931,6 +931,17 @@ impl<'a> Generator<'a, '_> {
             return Ok(());
         };
 
+        // Handle when this statement creates a new alias of a caller variable (or of another alias),
+        if let Target::Name(dstvar) = l.var {
+            if let Expr::Var(srcvar) = **val {
+                if let Some(caller_alias) = self.locals.get_caller(srcvar) {
```

Comment:
Since 1.88, `let_chain` feature is now stable, so we can now write:

```suggestion
        if let Target::Name(dstvar) = l.var
            && let Expr::Var(srcvar) = **val
            && let Some(caller_alias) = self.locals.get_caller(srcvar)
        {
```

Reply:
I specifically refrained from using let-chains because I didn't know which compiler version we were targeting. All the better :D

Reply:
You might need to update the MSRV and the version used in CI but otherwise, go for it!

Reply:
I have to change the rust edition to 2024 to use them, though. Is that okay?

Reply:
Yep, go for it! Next release will be a major one anyway so let's put as many things as possible haha.